packages/network-controller/src/NetworkController.ts

```diff
@@ -629,7 +629,7 @@ export class NetworkController extends BaseControllerV2<
    *
    * @returns The list of known network clients.
    */
-  getNetworkClientsById(): AutoManagedBuiltInNetworkClientRegistry &
+  getNetworkClients(): AutoManagedBuiltInNetworkClientRegistry &
```

Comment:
As discussed on Slack, if you assign this to a variable, but then you want an array version of this, the new name here forces you to name that array variable something else other than `networkClients`, which might be the more obvious choice. To save an extra decision from having to be made, what do we think about naming this something like `getNetworkClientRegistry`?

Reply:
sure I think that makes some sense!

Reply:
done [here](https://github.com/MetaMask/core/pull/1562/commits/c7be0e779acf5efb32ae102e8651e33d8a0febfc)